.github/workflows/test.yml

```diff
@@ -82,8 +82,8 @@ jobs:
         uses: actions-rs/tarpaulin@v0.1
         with:
           version: '0.9.0'
-          timeout: 300 # 5 mins for test coverage
-          args: '--ignore-tests -- --test-threads 1'
+          timeout: 600 # 10 mins for test coverage
```

Comment:
@Enrico2 let's go ahead and comment out the tarpaulin job all together and turn back on the normal linux one since code coverage isn't working well right now anyway

Reply:
sgtm!